.pre-commit-config.yaml

```diff
@@ -4,6 +4,7 @@
 exclude: >
     (?x)^(
       \.vscode/settings\.json|
+      docs/conf\.py|
```

Comment:
Should the new pre-commit hook be in `pre-commit-config.yaml`, since we are now checking for it in CI/CD? I was running `pre-commit run --all-files` locally, but this wasn't generating the new `conf.py` file in `docs/` which was then tripping up the pre-commit job.

Reply:
You can’t put it in the pre-commit, because you’d have to dynamically install the package from this repo (it like a cyclic dependency), that’s why pre-commit has the separate hook test command

Reply:
`pre-commit try-repo -v -a --show-diff-on-failure`